tests/chart/test_auth_sidecar.py

```diff
@@ -8,13 +8,13 @@
 from . import get_containers_by_name
 
 
-def common_dagserver_sts_test_cases(docs, docs_length):
+def common_sts_test_cases(docs, docs_length, release_name, kind):
```

Comment:
This applies to more than just `sts`
```suggestion
def common_pod_manager_test_cases(docs, docs_length, release_name, kind):
```